documentation/wiki/Mac-Debugging.md

```diff
@@ -0,0 +1,38 @@
+#Debugging with MacOS
+Say unit tests are failing from Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64
+* Open terminal 
+  * Hit command-space, type terminal, hit enter. Alternatively, hit magnifying glass (spotlight) in upper-right corner and search for terminal.)
+* Build something
+  * Navigation in terminal is similar to command prompt (cd), although you type `ls` in place of `dir`.
+  * **Use ./build.sh instead of .\build.cmd.**
```

Comment:
Do you think it's valuable to have this level of OS documentation here? We tend to assume command-line skills and focus on what's unique(ish) to this repo.

Reply:
I do, primarily because the audience for this document includes people who have lived their entire lives in the Microsoft ecosystem. I can imagine these sorts of small differences derailing someone for fifteen minutes as they look for something very specific that doesn't exist on Macs. I don't think it will really bother anyone.